lms/envs/common.py

```diff
@@ -2596,6 +2596,8 @@
         'language_proficiencies',
         'bio',
         'account_privacy',
+        # Not an actual field, but used to signal whether badges should be public.
```

Comment:
No. It is not stored in the user's preferences or on any model. It is purely configuration.